docs/best-practices.md

```diff
@@ -12,6 +12,9 @@ When adding an entity to the configuration file, use PascalCasing, so that the g
   - `updateCompositeNameEntity`
   - `deleteCompositeNameEntity`
 
+- Stored Procedures
```

Comment:
If the entity is backed by a stored procedure, both query or mutation would be named:
 - `executeCompositeNameEntity`